ArjunaJTS/orbportability/pom.xml

```diff
@@ -24,7 +24,7 @@
   <description>orb portability harness</description>
   <packaging>jar</packaging>
   <properties>
-    <openjdk-orb.jar>${project.build.directory}/openjdk-orb-8.0.2.Final.jar</openjdk-orb.jar>
+    <openjdk-orb.jar>${project.build.directory}/openjdk-orb-8.0.4.Final.jar</openjdk-orb.jar>
```

Comment:
Maybe this could be ${project.build.directory}/openjdk-orb-${version.org.jboss.openjdk-orb}.jar